github-sample/.github/workflows/logicapp_container_deploy.yml

```diff
@@ -0,0 +1,57 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/azure/functions-action
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: (container) Build and Deploy Logic App. 
+
+env:
+  RG_CON: 'your-con-rg'
+  LA_NAME: 'your-la-name'
+  IMAGE_NAME: 'your-image-name'
+  IMAGE_TAG: 'your-tag'
```

Comment:
I would suggest using the build id of the pipeline run as the image tag, but not a blocker